content/minimal_example/fhir/example.fhir.json

```diff
@@ -1,22 +1,18 @@
 {
```

Comment:
## Results: output updates - `example.fhir.json`
@ShahimEssaid There are 1-2 other example content 'modules' that I still have to convert. For now, I've just converted this one.

Raw files: [Archive.zip](https://github.com/timsbiomed/sssom-on-fhir-content/files/15313914/Archive.zip)
- This contains the input TSV, and the output from before this PR, and the output now.

Differences you can see in the diff:
- Bug fix: `ValueString` --> `valueString`
- Bug fix: Empty fields: Previously, some fields were automatically being populated with empty strings, e.g. `      "source": "",`
- Delete: `title`: This now comes from the SSSOM `title` field, which doesn't exist in the input TSV. So it was removed.
- New bug revealed: Inconsistent ID / url / name: These are generate based on a UUID hash that SSSOM assigns. Even though the input has not changed, the UUID changed. I don't know if I'll be able to get `sssom-py` to fix this, so the user will want to assign their own `mapping_set_id` to alleviate this.